AzureMonitorAgent/packaging.sh

```diff
@@ -43,6 +43,9 @@ cp $input_path/MetricsExtension MetricsExtensionBin/
 # copy just the source of python-future
 cp -r ext/python-future/src/* ext/future
 
+# make the shim.sh file executable
+chmod +x shim.sh
```

Comment:
fwiw I am not sure this is necessary ... it is not done for omsagent

Reply:
hmm, I am not sure how else it would be made executable? OMS is also not using the "./" , so the way they're called is different maybe

Reply:
is it not working if you don't add that? 
i looked at waagent code, seems they are calling `Popen`, and using `shell=True`. so it should work fine without `chmod`

Reply:
No, it isn't running without it. The default perms are 644, so I just tested by changing shim.sh back to 644 and running waagent restart (which calls enable each restart) and it failed with permission denied. It will need the executable permission to run.